### ISSUE TITLE
cert override with mixed active content

### ISSUE THREAD

**Issue by TanviHacks** [lgarron/badssl.com#142]
It would be great to have a test case that has mixed active content and uses a certificate that requires a certificate override.  Thanks!

**Comment by lgarron** [comment]
Like #140?

**Comment by TanviHacks** [comment]
Not quite.  The top level page should have a bad cert (ex: expired, self-signed, wrong host) that the user has to add an exception for.  Then the loaded page should request an http script subresource.

**Comment by marumari** [comment]
Based on conversations in #150:

* `mixed.expired.badssl.com`
* `mixed-script.expired.badssl.com`

**Comment by lgarron** [comment]
Note that you can already visit:

- https://expired.badssl.com/mixed/image/
- https://expired.badssl.com/mixed/script/

**Comment by lgarron** [comment]
I'd like to avoid a combinatorial explosion of transport errors and content errors, so I would actually prefer to `wont-fix` this in favor of the URLs in the previous comment.

**Comment by marumari** [comment]
I'd be down with that.  @TanviHacks, does this work for you?
